heartbeat/process/sender.go

```diff
@@ -27,6 +28,7 @@ type ArgHeartbeatSender struct {
 	NodeDisplayName      string
 	KeyBaseIdentity      string
 	HardforkTrigger      heartbeat.HardforkTrigger
+	ChainHandler         data.ChainHandler
```

Comment:
Is the entire Chainhandler necessary? It could be a ChainHandler defined in the heartbeat package with just 1 method, GetCurrentBlockHeader that is needed. The mock would also be smaller.

Reply:
Done, used a new interface with just 2 methods

Reply:
ok